src/storageDataGroup.js

```diff
@@ -161,7 +161,7 @@ function readDataGroup(path) {
 function outputToObject(output) {
     const jsonString = output
         .replace(/ltm data-group internal .*? {/, '{')
-        .replace(/(\s*)(.*?) {/g, '$1"$2" : {')
+        .replace(/(\s*)("?)(.*?)("?) {/g, '$1"$3" : {')
```

Comment:
Why was this changed to $3?

Reply:
It's because the $3 is referencing the ().

Reply:
Yup, and $2 is now the match of zero or one double-quotes that should be before the name.